CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+# 1.2.7 (2021-01-19)
```

Comment:
Don't forget the upcoming merge for edgeAgent & edgeHub image update for the security patch.